R/install.R

```diff
@@ -1,4 +1,4 @@
-BINARY_BASE_URL <- "https://bioconductor.org/packages/%s/container-binaries%s"
+BINARY_BASE_URL <- "https://bioconductor.org/packages/%s/container-binaries/%s"
```

Comment:
If we are changing the default URL to include `bioconductor_docker` we should always have a `/` here. 
Currently we use `https://bioconductor.org/packages/3.15/container-binaries` as the base URL for bioconductor_docker binaries.

Reply:
Just so that we're on the same page, can you provide the explict URLs we're expecting to resolve?

Reply:
Based on your suggestion here https://github.com/Bioconductor/AnVIL/issues/47#issuecomment-1019000817 
The links should be `https://bioconductor.org/packages/<bioc-version>/container-binaries/<container>/` which would resolve to (AFAICT): 

```
https://bioconductor.org/packages/3.15/container-binaries/anvil-rstudio-bioconductor/src/contrib/PACKAGES.gz
https://bioconductor.org/packages/3.15/container-binaries/bioconductor_docker/src/contrib/PACKAGES.gz
```